server/services/licenceService.js

```diff
@@ -62,11 +62,11 @@ module.exports = function createLicenceService(licenceClient, establishmentsClie
     }
 
     function markForHandover(nomisId, sender, receiver) {
-        if(!licenceStates[sender] || !licenceStates[sender][receiver]) {
+        if(!transitions[sender] || !transitions[sender][receiver]) {
```

Comment:
maybe use the util getIn 
`getIn(transitions, [sender, reciever])` 

saves you from having to do 
`!transitions[sender] || !transitions[sender][receiver]`

Reply:
Yep, that is nicer. Ta